network/requestTracker.go

```diff
@@ -39,9 +42,9 @@ const (
 // TrackerRequest hold the tracking data associated with a single request.
 type TrackerRequest struct {
 	created            time.Time
-	remoteHost         string
+	remoteHost         string // changed by updateRequestRemoteAddr if UseXForwardedForAddressField is set
 	remotePort         string
-	remoteAddr         string
+	remoteAddr         string // not changed by updateRequestRemoteAddr. TODO FIXME
```

Comment:
What is the FIXME that remains? The way I read it, we are keeping track of the original address here, and the translated address above.  Is that bad?

Reply:
yes, this is an original address, I left this comment to address in followup PR #5851